composer.json

```diff
@@ -14,9 +14,9 @@
     "require-dev": {
         "editorconfig-checker/editorconfig-checker": "^10.2.3",
         "ergebnis/composer-normalize": "^2.28",
-        "phpstan/phpstan": "^0.12.96",
-        "phpstan/phpstan-phpunit": "^0.12.22",
-        "phpstan/phpstan-strict-rules": "^0.12.11",
+        "phpstan/phpstan": "^1.7.14",
```

Comment:
There is no reason not to use fixed versions of dev requirements, especially in the case of tools for static analysis where even minor versions regularly break the pipeline.

Reply:
That is questionable, breaking pipeline lets you know there is some issue with your code, but blocks the unrelated MR. Lets keep it this way, it helped me for example here: https://github.com/shipmonk-rnd/phpstan-rules/pull/10